recipes/nb_conda_kernels/run_test.sh

```diff
@@ -0,0 +1,6 @@
+cd "${SRC_DIR}"
+
+conda install r-irkernel -y -n _test -c r
```

Comment:
This does really boil down to the difference between unit/integration testing and system testing.  If these tests are to confirm that all features of `nb_conda_kernels` are still functioning as they are in the source reporitory then we should also be using a testing matrix which exclude `r` and `python` systematically. However, if these tests are to give us confidence that `nb_conda_kernels` is working as expected, I can comfortably imagine a situation where `r` isn't needed for testing.

I completely accept the whole thing is a big thick gray line, but am keen to get into the mindset of ensuring we have enough confidence in our builds that they correctly represent the testing taking place in the development repository, without necessarily having to run the whole test suite (and its full dependency matrix) to give us that confidence.

In some situations the full test suite is so light that is makes no sense to bother worrying about the distinction, but in many cases, that simply isn't an option.

xref: https://github.com/conda-forge/notebook-feedstock/pull/6 for a similar conversation